common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -101,7 +101,7 @@
                 el: this.$('.inline-threads'),
                 collection: self.discussion,
                 courseSettings: self.course_settings,
-                hideRefineBar: true  // TODO: re-enable the search/filter bar when it works correctly
+                hideRefineBar: false  // TODO: re-enable the search/filter bar when it works correctly
```

Comment:
Do we really need to enable search/filter bar? It is not visible in current Prod/QA environment we should keep it hidden.